app/views/shared/_navbar.html.slim

```diff
@@ -41,8 +41,8 @@ nav.navbar.navbar-expand-custom.sticky-top
               li
                 = link_to t("nav_item.sign_in"), new_user_session_path, class: "dropdown-item profile"
             - else
-              - if current_user.admin?
+              - if current_user.admin? || current_user.moderator?
```

Comment:
Add method in User model `.staff_members` for this code and replace everywhere such code with this method